projects/plugins/jetpack/extensions/blocks/latex/edit.js

```diff
@@ -0,0 +1,110 @@
+import { useBlockProps } from '@wordpress/block-editor';
+import {
+	Button,
+	TextareaControl,
+	/* eslint-disable @wordpress/no-unsafe-wp-apis */
+	__experimentalHStack as HStack,
+	__experimentalSurface as Surface,
+	__experimentalVStack as VStack,
+	/* eslint-enable @wordpress/no-unsafe-wp-apis */
+	Placeholder,
+} from '@wordpress/components';
+import { useDispatch } from '@wordpress/data';
+import { useState, useEffect, useRef } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+import katex from 'katex';
+import { icon } from './icon';
+import 'katex/dist/katex.min.css';
+import './editor.css';
+
+const LatexPlaceholder = () => {
+	return (
+		<span className="jetpack-latex-rendered-placeholder">{ __( 'Write LaTex…', 'jetpack' ) }</span>
+	);
+};
+
+export default function Edit( { attributes, setAttributes, isSelected } ) {
+	const { latex = '' } = attributes;
+	const textareaRef = useRef();
+	const [ renderAreaRef, setRenderAreaRef ] = useState( null );
+	const dispatch = useDispatch();
+	const props = useBlockProps( { className: 'jetpack-latex' } );
```

Comment:
Is the explicit className here necessary? Isn't it added there by default since it's the block name?


```suggestion
	const props = useBlockProps();
```